### DISCUSSION TITLE
Is there any hope if there will be support for .CLIP file format?

### DISCUSSION THREAD

**Discussion by towekki** [QL-Win/QuickLook#1472] [Q&A]
I don't know how often people make plugins for QuickLook, but it would be awesome if the .clip file format could be viewed the same way .psds do. Clip Studio Paint is a very popular program so I was surprised that its file format wasn't already supported in this program!

**Comment by emako** [comment]
Note: May we can develop a new plugin using https://github.com/unai-d/unclip support

**Reply by towekki** [reply]
Thank you for the update! I'll be looking forward to any new developments

**Reply by emako** [reply]
> Thank you for the update! I'll be looking forward to any new developments

Actually, `.clip` file is not like a regular image file. 
A `.clip` file contains multiple image layers, which is not a display effect currently supported by QuickLook.
